app.js

```diff
@@ -2,7 +2,7 @@ const express = require("express");
 const app = express();
 const { getTopics } = require("./controllers/topics.controllers");
 const endpoints = require("./endpoints.json");
-const { getArticleById, getArticles } = require("./controllers/articles.controllers");
+const { getArticleById, getArticles, getCommentsByArticleId } = require("./controllers/articles.controllers");
 
 app.use(express.json());
```

Comment:
This needs to be removed, as you do not have a post/patch request yet! 🚨  See the notes if you're unsure what I mean by this.